infrastructure/labelmaker/labelmaker.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Can you remove "All Rights Reserved."  
The header should just be "# Copyright 2017 Google Inc."

Thanks